doc/source/tune/_tutorials/tune-serve-integration-mnist.py

```diff
@@ -0,0 +1,645 @@
+# flake8: noqa
+"""
+Model selection and serving with Ray Tune and Ray Serve
+=======================================================
```

Comment:
How about let's start the first sentence with the goal of this document? Or rather, what you're trying to solve?